training-code/text-classification/Readme.md

```diff
@@ -8,3 +8,7 @@ conda install -c conda-forge minio
 docker build -t intent-classification .
 docker run  -e MINIO_SERVER="172.17.0.3:9000" intent-classification
 <TODO>, currently minIO container access will fail
+
+docker run --name trainer1 --hostname=trainer1 -p 12356:12356 -e MASTER_ADDR="trainer1" -e WORLD_SIZE=2 -e RANK=0 -e MASTER_PORT=12356 -it localhost:3000/orca3/intent-classification
+
+docker run --name trainer2 --hostname=trainer2  -e MASTER_ADDR="trainer1" -e WORLD_SIZE="2" -e RANK=1 -e MASTER_PORT=12356 -it localhost:3000/orca3/intent-classification
```

Comment:
Docker run will fail at the communication group setup phase. docker trainer2 couldn't reach docker trainer1.

Traceback (most recent call last):
  File "/opt/intent-classification/train.py", line 165, in <module>
    dist.init_process_group("gloo", rank=RANK, world_size=WORLD_SIZE)
  File "/opt/conda/lib/python3.7/site-packages/torch/distributed/distributed_c10d.py", line 520, in init_process_group
    store, rank, world_size = next(rendezvous_iterator)
  File "/opt/conda/lib/python3.7/site-packages/torch/distributed/rendezvous.py", line 199, in _env_rendezvous_handler
    store = TCPStore(master_addr, master_port, world_size, start_daemon, timeout)
ValueError: host not found: Name or service not known